src/styles/_person.scss

```
@@ -0,0 +1,41 @@
+.person {
+  text-align: center;
+
+  &__photo {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+  }
+
+  &__name {
+    margin-top: 12px;
+    font-weight: 700;
+  }
+
+  &__position {
+    margin: 0;
+    color: #696973;
```

такі кольори краще виносити в окремі змінні
```suggestion
    color: $some-color;
```